terraform/digitalocean/k8s/terraform.tf

```diff
@@ -3,6 +3,12 @@ terraform {
   required_version = ">= 0.12.0"
 
   backend "s3" {
+    region = "us-east-1"
+    access_key = ""
+    secret_key = ""
+    bucket = ""
+    key = ""
+    endpoint = ""
```

Comment:
Can we not specify empty params and region? Or DO requires it?

Reply:
Comit as it would be refactored